src/d/actor/d_a_mmusic.cpp

```diff
@@ -32,13 +51,22 @@ s32 daMmusic::Act_c::_create() {
 }
 
 /* 00000268-000002B4       .text _delete__Q28daMmusic5Act_cFv */
-BOOL daMmusic::Act_c::_delete() {
+bool daMmusic::Act_c::_delete() {
     /* Nonmatching */
+    int iVar1;
+    if (this->field_0x2A0 != 0) {
+        *(unsigned int*)(iVar1 + 0x60) = 0xffffffff;  // should be an object, modify the field
+        *(unsigned int*)(iVar1 + 0x20c) = *(unsigned int*)(iVar1 + 0x20c) | 1;
+        *(unsigned int*)(field_0x2A0) = 0;
+    }
+    delete_se();
+    return true;
 }
 
 /* 000002B4-000002C0       .text init_se__Q28daMmusic5Act_cFv */
 void daMmusic::Act_c::init_se() {
     /* Nonmatching */
+    (this->field_0x2D4) = 0x78;
```

Comment:
What's not matching about this? Given how the value is 0x78 / 120 (4 seconds at 30fps), I'm guessing this is a timer.